libs/stac/docs/stac-vs-odc.rst

```diff
@@ -0,0 +1,85 @@
+STAC vs Open Datacube
+#####################
+
+`Open Datacube`_ project on which this library is based started before `STAC`_
+spec existed. As a result ODC uses different terminology for otherwise very
+similar concepts.
```

Comment:
```suggestion
The `Open Datacube`_ (ODC) project, on which this library is based, started before `STAC`_
spec existed. As a result ODC uses different terminology for otherwise very
similar concepts.
```